src/libgpu/src/vulkan/vulkan_driver.h

```diff
@@ -601,12 +601,12 @@ class Driver : public gpu::VulkanDriver, public Pm4Processor
    void _invalidateSurface(SurfaceObject *surface, SurfaceSubRange range);
    void _barrierSurface(SurfaceObject *surface, ResourceUsage usage, vk::ImageLayout layout, SurfaceSubRange range);
    SurfaceObject * getSurface(const SurfaceDesc& info);
-   void transitionSurface(SurfaceObject *surface, ResourceUsage usage, vk::ImageLayout layout, SurfaceSubRange range, bool skipChangeCheck = false);
+   void transitionSurface(SurfaceObject *surface, ResourceUsage usage, vk::ImageLayout layout, SurfaceSubRange range, bool doChangeCheck = true);
```

Comment:
IMO the "do" here might be unecessary to have?

Reply:
I don't know... `changeCheck` could be understands as "change the check".

Reply:
What exactly is it doing then?

Reply:
It checks the surface has changed.

Reply:
`do` is fairly unconventional for bool names in my experience. Perhaps `checkSurfaceChanges`.

Reply:
I agree, `checkSurfaceChanges` is better.